### DISCUSSION TITLE
Migrating to new host

### DISCUSSION THREAD

**Discussion by tehkurt** [posativ/isso#930] [Q&A] [answered]
Hello self-hosted comment enthusiasts. I am inquiring on the best way to move my isso installation to a new host. I have set up a new host, installed isso, and moved my db file to the new host. Everything fired up and seems to be working as expected, except the first comment left on any post shows twice, once with gravatar (correct) and once without. 

Is there any documentation on the correct method for doing an Isso transplant to a new host? 

![2022-10-17 21_11_31-Waiting—Value or Waste_](https://user-images.githubusercontent.com/15126901/196312031-a19e8cd2-666e-4a89-9649-f2e25eb18b9c.png)

**Comment by ix5** [comment, marked as answer]
To better help debug your problem, please upload the following information:
- Version of Isso you're using
- Operating system and Python version
- What happened? What's wrong?
- How can one reproduce your issue?
- Config file and client-side embed script
- Any (relevant) log outputs

**Reply by tehkurt** [reply]
I gathered all requested info and then it hit me. I had been testing with adding the script through Ghost's code injection feature and it ended up being there and in the post.hbs file that controls post layout. Removing the script tag from the code injection settings resolved my issue. I will leave all the relevant info below for posterity's sake. Thanks for helping me work through this!
----------------------------------------------------------------------------------------------------------------------------------------

Isso v 0.13.0
Ubuntu 22.04.1

I migrated to a new host VM and have a fresh install of isso using my existing config and database. I run isso on a self hosted Ghost blog that is authored by my wife's Grandmother. As seen in the previous screenshot, the first comment left on a post is showing up twice. If the oldest comment gets deleted, the new oldest comment becomes doubled, I have tested starting off with a new database file on the new host as well with the same result.

This issue can be reproduced by being the first comment on any post
